modules/ROOT/Edits/modify-partner-settings.adoc

```diff
@@ -0,0 +1,62 @@
+= Modify a Partner's Settings
+
+You can modify a partner's settings including contact details, identifiers, digital certificates, acknowledgements, control numbers, and more.
```

Comment:
```suggestion
You can modify partner settings such as contact details, identifiers, digital certificates, acknowledgments, and control numbers.
```

Reply:
Be sure that the subtopics for each follow the order in which they are introduced here.